### PR TITLE
Add button to open database config from the new databases UI

### PR DESCRIPTION
**Pull request by shati-patel**
Adds a UI icon in the new databases panel to open the `workspace-databases.json` config file.

pencil icon to open database config file

~Note: I couldn't quite decide whether to call the command `codeQLDatabases.openConfigFile` or `codeQLDatabasesExperimental.openConfigFile`, so I've gone with the former for now. (Which saves us from adding a case to the [command-lint](https://github.com/github/vscode-codeql/blob/93054e14a27f2001dac4b6230fe2b47870d31aa6/extensions/ql-vscode/test/pure-tests/command-lint.test.ts#L30-L53) test.) Happy to switch over again though~ 🤷🏽 😅  EDIT: switched to `codeQLDatabasesExperimental`.

See internal linked issue for more details!

## Checklist

N/A - internal only 📝 

- [ ] [CHANGELOG.md](https://github.com/github/vscode-codeql/blob/main/extensions/ql-vscode/CHANGELOG.md) has been updated to incorporate all user visible changes made by this pull request.
- [ ] Issues have been created for any UI or other user-facing changes made by this pull request.
- [ ] _[Maintainers only]_ If this pull request makes user-facing changes that require documentation changes, open a corresponding docs pull request in the [github/codeql](https://github.com/github/codeql/tree/main/docs/codeql/codeql-for-visual-studio-code) repo and add the `ready-for-doc-review` label there.

### CODE REVIEW

File: extensions/ql-vscode/package.json [hunk 1]
```diff
@@ -354,6 +354,14 @@
         "command": "codeQL.copyVersion",
         "title": "CodeQL: Copy Version Information"
       },
+      {
+        "command": "codeQLDatabases.openConfigFile",
```
Comment (charisk) [inline extensions/ql-vscode/package.json:358]: It'd be good to keep the two database panels and their commands completely separate for now and go with `codeQLDatabasesExperimental`

File: extensions/ql-vscode/package.json [hunk 2]
```diff
@@ -970,6 +983,10 @@
           "command": "codeQL.chooseDatabaseLgtm",
           "when": "config.codeQL.canary"
         },
+        {
+          "command": "codeQLDatabases.openConfigFile",
```
Comment (charisk) [inline extensions/ql-vscode/package.json:987]: Sorry for the ignorance, but why do we need to do this?

Comment (shati-patel) [reply extensions/ql-vscode/package.json:987]: This stops the command showing up in the command palette, so it's only accessible from the Databases panel. 

I guess we could make it more widely visible later, but I didn't see any reason to do that yet 🤔

File: extensions/ql-vscode/package.json (hunk 1 above)
Comment (shati-patel) [reply extensions/ql-vscode/package.json]: Done in https://github.com/github/vscode-codeql/pull/1719/commits/c7c0ad49cf4e665df256daf702cb6f5fa0e05894 (and https://github.com/github/vscode-codeql/pull/1719/commits/c7b217c8fa35a62a0f0bd80a06e1a002d6e42311)

File: extensions/ql-vscode/package.json (hunk 2 above)
Comment (charisk) [reply extensions/ql-vscode/package.json]: Oh ofc! Sorry I forget that these commands are automatically added to the command pallet.

**Review by charisk — APPROVED** [review]
LGTM, thanks!

**Pull request merged**